mints/02.json

```diff
@@ -0,0 +1 @@
+{"name":"contract/valory/conditional_tokens:0.1.0","description":"The scaffold contract scaffolds a contract to be implemented by the developer.","code_uri":"ipfs://bafybeihufvbfz7l7fb3fvsalp6jmplkxoc3rs3b25apultqmoelrvvx2de","image":"ipfs://bafybeif2uk24bpv55cxzjwwifqp7rxhzrveg2dp2ylg2x24pswy4tjv7iy","attributes":[{"trait_type":"version","value":"0.1.0"}]}
```

Comment:
"The scaffold contract scaffolds a contract to be implemented by the developer." -> you need to update the metadata of this package in its yaml @jmoreira-valory